modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionTypeThemeCSS/client-extension/theme-css-project-create-config-file-frontend-token-definition/client-extension.yaml

```diff
@@ -0,0 +1,4 @@
+theme-css-project-create-config-file-frontend-token-definition:
```

Comment:
Please update the folder structure to match below:

modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionTypeThemeCSS/theme-css-with-frontend-token-definition-json/client-extension.yaml

modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionTypeThemeCSS/theme-css-without-frontend-token-definition-json/client-extension.yaml

Reply:
Hey @algobob the folder structure still needs a small change to match. You have an extra `client-extension` directory that's not needed. Can you remove that? Then you will have the same structure I listed above.

Reply:
oh my bad, i missed it. I'm looking for too long this piece of code haha